elasticdl/python/pserver/embedding_table.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import tensorflow as tf
+
+
+class EmbeddingTable(object):
+    def __init__(self, name, dim=None, initializer=None):
+        self.name = name
+        self.dim = dim
+        self.initializer = initializer
+        self.embedding_vectors = {}
+
+    def get(self, indices):
+        if len(indices) == 0:
+            return None
+        values = []
+        for i in indices:
+            if i not in self.embedding_vectors:
```

Comment:
Maybe better change the code snippet 1 to code snippet 2 to query dictionary only once if `i in embedding_vectors`. Because querying dictionary is not fast when there are many embedding vectors.

code snippet 1:
```python
if i not in self.embedding_vectors:
    ...
else:
   value = self.embedding_vectors[i]
``` 

code snippet 2:
```python
value = self.embedding_vectors.get(i, None)
if value is None:
    ....
```